### ISSUE TITLE
Import route_directions.txt

### ISSUE THREAD

**Issue by mharvey** [tulsawebdevs/django-multi-gtfs#63]
I understand that this is a non-standard extension to GTFS, but it seems to be adopted by several agencies (Mainly TriMet, but others are including it as well) and the information within is extremely useful. [Here](http://transitfeeds.com/p/trimet/43/latest/file/route_directions.txt) is a sample.

Is there opposition to including this as something that would be imported if it's present in the archive abut ignored if it's not?

Looking at the code, I'd be happy to take a stab at implementing this and send a PR for review.

**Comment by jwhitlock** [comment]
Yes, I'd look at the PR. That seems like a reasonable optional extension.
